rnacentral_pipeline/rnacentral/ftp_export/coordinates/data.py

```diff
@@ -76,6 +78,9 @@ def build(cls, index, raw):
             "databases": clean_databases(raw["databases"]),
         }
         if not metadata["providing_databases"]:
+            if not raw["was_mapped"]:
+                print(raw)
```

Comment:
Probably better to change the exception to f".. {raw}" to show what data has the
issue.